config/config.php

```diff
@@ -1,33 +1,33 @@
 <?php
 
-use Zend\Stdlib\ArrayUtils;
-use Zend\Stdlib\Glob;
+use Zend\ConfigAggregator\ConfigAggregator;
+use Zend\ConfigAggregator\PhpFileProvider;
 
-/**
- * Configuration files are loaded in a specific order. First ``global.php``, then ``*.global.php``.
- * then ``local.php`` and finally ``*.local.php``. This way local settings overwrite global settings.
- *
- * The configuration can be cached. This can be done by setting ``config_cache_enabled`` to ``true``.
- *
- * Obviously, if you use closures in your config you can't cache it.
- */
+// To enable or disable caching, set the `ConfigAggregator::ENABLE_CACHE` boolean in
+// `config/autoload/local.php`.
+$cacheConfig = [
+    'config_cache_path' => 'data/config-cache.php',
+];
 
-$cachedConfigFile = 'data/cache/app_config.php';
+$aggregator = new ConfigAggregator([
+    //Include cache config for zf-development-mode
+    function () use ($cacheConfig) {
+        return $cacheConfig;
+    },
```

Comment:
This is required to get the config cache path into zf-development-mode. I cannot find a cleaner way to pass the path to zf-dev-mode and I did not want to hardcode the path either. (https://github.com/zfcampus/zf-development-mode/blob/master/src/ConfigDiscoveryTrait.php#L60-L62)

This closure hack is because `ConfigAggregator` does not accept a plain array, it requires a closure.